Source/OIDTokenRequest.m

```diff
@@ -273,6 +275,13 @@ - (NSURLRequest *)URLRequest {
   OIDURLQueryComponent *bodyParameters = [self tokenRequestBody];
   NSMutableDictionary *httpHeaders = [[NSMutableDictionary alloc] init];
 
+  /**
+   * GitHub will only return a spec-compliant response if JSON is explicitly defined
```

Comment:
As a rule, AppAuth doesn't support special casing for non-spec compliant implementations of RFC 6749. We need to take a strong stance on this, to avoid bloating the library – and also, there is no reason for people *not* to support RFC 6749 correctly.

That said, since the response should be JSON, I don't mind sending this header. Please change the comment so that the justification isn't something that goes against our policies :-)